grafonnet/grafana.libsonnet

```diff
@@ -27,4 +27,5 @@
   gaugePanel:: import 'gauge_panel.libsonnet',
   barGaugePanel:: import 'bar_gauge_panel.libsonnet',
   statPanel:: import 'stat_panel.libsonnet',
+  stackdriver:: import 'stackdriver.libsonnet',
```

Comment:
looks like the official name is now Google Cloud Monitoring, and internally the plugin is called `cloud-monitoring`.

Reply:
https://github.com/grafana/grafana/blob/aae4ca754497ebfdbf70560596cd9185fd787b21/public/app/plugins/datasource/cloud-monitoring/plugin.json